pkg/manager/member/pd_member_manager.go

```diff
@@ -267,9 +267,10 @@ func (m *pdMemberManager) shouldRecover(tc *v1alpha1.TidbCluster) bool {
 		if !podutil.IsPodReady(pod) {
 			return false
 		}
-		status, ok := tc.Status.PD.Members[pod.Name]
-		if !ok || !status.Health {
-			return false
+		for pdName, pdMember := range tc.Status.PD.Members {
```

Comment:
What if there is no member of this Pod?

Reply:
If there is no member of this pod, there is no need to set off the failover function. When a member is reported as unhealthy, the controller begins to do failover operations.

Reply:
It can be the launching stage.